bin/rust-server-petstore.sh

```diff
@@ -38,4 +38,8 @@ for spec_path in modules/openapi-generator/src/test/resources/*/rust-server/* ;
 		 $@"
 
   java $JAVA_OPTS -jar $executable $args
+
+  if [ $? -ne 0 ]; then
+    exit $?
+  fi
```

Comment:
Good catch. Any reason to not just `set -e`, though?

Reply:
Opinion is divided on whether `set -e` is a good idea - see http://mywiki.wooledge.org/BashFAQ/105 for some opinions.